views/index.erb

```diff
@@ -1,14 +1,33 @@
 <% if @profile %>
-  <h3><%= @profile["first_name"] %>'s Amazing Profile</h3>
-
+  <h1><%= @profile["first_name"] %>'s Amazing Profile</h1>
+  <!-- Add picture -->
+  <div id="profile-pic">
+    <img src="../images/profile_pic.jpg" width="150" height="auto" border="0" alt="Natascha's profile picture" />
+  </div>
   <!-- TODO- Perhaps re-jig this to fit your new design / layout -->
-  <% @profile.each do |field, value| %>
-    <div class="profile-section box">
-      <h4><%= field %></h4>
-      <p class="profile-value"><%= value %></p>
+  <div id="profile-overview">
+    <div class="info">
+    <% @profile.each do |field, value| %>
+      <!-- all profiles should be stand-alone -->
+      <% if field != "codecademy" && field != "github" && field != "homepage" && field != "twitter" %>
```

Comment:
This logic feels a little clumsy to me, especially since it's repeated in line 23 below. Since we know `@profile` is a Ruby `Hash`, perhaps we could simplify this code by removing the conditionals and just indexing into `@profile` using the keys we know exist, instead of iterating and performing these checks every time...